src/infrastructure/content/NetworkMessageHandler.java

```diff
@@ -0,0 +1,151 @@
+package infrastructure.content;
+
+import networking.INotificationHandler;
+import org.json.JSONObject;
+import java.util.HashMap;
+import infrastructure.validation.logger.ILogger;
+import infrastructure.validation.logger.LogLevel;
+import infrastructure.validation.logger.LoggerFactory;
+import infrastructure.validation.logger.ModuleID;
+
+/**
+ * NetworkMessageHandler class implements the INotificationHandler interface.
+ * this includes one method i.e @onMessageReceived()
+ * this class is used to notify and to send data to the UI module 
+ * as new user joined or as a user exit or as message sent by a user
+ * 
+ * @author Talha Yaseen
+ */
+public class NetworkMessageHandler implements INotificationHandler {
+	
+	/**
+	 * Creating instance of ContentCommunicator class to access handler map and image map
+	 */
+	private ContentCommunicator cc = ContentFactory.getContentCommunicator();
+	
```

Comment:
The methods needed to access the variables you want(like the imagemap) has been made static. You do not require an object of CC to do it. You can access the static variables with the static methods